<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kristen ITC</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>511</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Mono</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>원하는 문구를 입력하세요</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans KR Medium</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>text</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
